test/utils/fizzy_engine.cpp

```diff
@@ -90,7 +90,7 @@ bool FizzyEngine::instantiate(bytes_view wasm_binary)
                         {"env", "adler32", {fizzy::ValType::i32, fizzy::ValType::i32},
                             fizzy::ValType::i32, env_adler32},
                     });
-        m_instance = fizzy::instantiate(module, imports);
+        m_instance = fizzy::instantiate(std::move(module), imports);
```

Comment:
This has no effect because the `module` is const.

Reply:
This is already done in #581 including making module non-const.

Reply:
I did this at the same time you did #581, and asked since if I can merge the other changes from this into #581, but no response.